contributing/testing-best-practices.md

```diff
@@ -0,0 +1,49 @@
+
+# Testing Best Practices
```

Comment:
do you mean for instance when describing variable MONGODB_ATLAS_PROJECT_ID, function ProjectIDExecution, etc. to link to some part in the code?

if that's the case I'm not sure I would do that because it's easy that links to code get outdated when we change the code/tests.

I think it's better that people just search for words like MONGODB_ATLAS_PROJECT_ID in the code to see how it can be used.

Reply:
I left a couple of cases below of where to leave an example.

> it's easy that links to code get outdated when we change the code/tests.
Not properly, you can make a link to a file by fixing the commit. Also if an example ever gets outdated it means also the guideline is outdated, so it doesn't really create an extra effort.

Rationale is to think: how can I speed up my team to implement the right practices and "see" what I mean?

Reply:
adding examples